pom.xml

```diff
@@ -92,6 +92,9 @@
              Force bigger region size in attempt to help G1 utilize heap fully. -->
         <air.test.jvm.additional-arguments>
             -XX:G1HeapRegionSize=32M
+            -XX:+UnlockDiagnosticVMOptions
```

Comment:
is it required to set XX:GCLockerRetryAllocationCount ?

Reply:
It's actually needed for both :smile: